tests/containers/containerd_rmt.pm

```diff
@@ -29,12 +29,14 @@ sub run {
     install_kubectl();
     install_helm();
 
-    my $rmt_helm = get_var('RMTTEST_REPO', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
+    my $rmt_helm = get_var('RMTTEST_CHART', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
     # pull in the testsuite
     assert_script_run("curl -sSLk $rmt_helm | tar -zxf -");
     my $rmt_config = get_var('RMTTEST_CONFIG', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
     assert_script_run("curl -sSLOk $rmt_config");
-    assert_script_run("helm install rmt ./helm-charts-main/rmt-helm -f myvalue.yaml");
+    my $rmt_repo = get_var('RMTTEST_REPO', 'registry.suse.de/suse/sle-15-sp5/update/cr/totest/images/suse/rmt-server');
```

Comment:
Since this repo is containing a hardcoded version, I would make this a `get_required_var` instead, to prevent silent "we missed this version update" issues.